### PR TITLE
Rename all remaining occurences of carrier to fuseml

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ What **FuseML** is **NOT**:
 
 ## Inception and Direction
 
-FuseML originated as a fork of our sister open source project [Carrier](https://github.com/SUSE/carrier), a lightweight open source PaaS built on top of Kubernetes, then has been gradually transformed and infused with the MLOps concepts that make it the AI/ML orchestration tool that it is today.
+FuseML originated as a fork of our sister open source project [Fuseml](https://github.com/SUSE/fuseml), a lightweight open source PaaS built on top of Kubernetes, then has been gradually transformed and infused with the MLOps concepts that make it the AI/ML orchestration tool that it is today.
```
Comment (stefannica) [inline README.md:46]: Eager rename :)

File: docs/wordpress.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Installing Wordpress with Carrier
```
Comment (stefannica) [inline docs/wordpress.md:1]: I think we should remove this file. It's no longer accurate (or is it ?).

File: kubernetes/cluster.go [hunk 3]
```diff
@@ -34,13 +34,13 @@ import (
 )
 
 const (
-	// APISGroupName is the api name used for carrier
-	APISGroupName = "carrier.suse.org"
+	// APISGroupName is the api name used for fuseml
+	APISGroupName = "fuseml.suse.org"
```
Comment (stefannica) [inline kubernetes/cluster.go:38]: how about "fuse.ml" instead of "fuseml.suse.org" ?

File: deployments/tekton.go [hunk 4]
```diff
@@ -342,15 +342,15 @@ func applyTektonStaging(c *kubernetes.Cluster, ui *ui.UI) (string, error) {
...
+	return helpers.Kubectl(fmt.Sprintf("apply -n fuseml-workloads --filename %s", tmpFilePath))
 }
 
 func createTektonIngress(c *kubernetes.Cluster, subdomain string) error {
 	_, err := c.Kubectl.ExtensionsV1beta1().Ingresses("tekton-pipelines").Create(
 		context.Background(),
 		// TODO: Switch to networking v1 when we don't care about <1.18 clusters
 		// Like this (which has been reverted):
-		// https://github.com/SUSE/carrier/commit/7721d610fdf27a79be980af522783671d3ffc198
+		// https://github.com/SUSE/fuseml/commit/7721d610fdf27a79be980af522783671d3ffc198
```
Comment (stefannica) [inline deployments/tekton.go:353]: I would keep this as is. The new URL doesn't work.

**Review by stefannica — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jsuchome) [reply README.md:46]: ah

File: docs/wordpress.md (hunk 2 above)
Comment (jsuchome) [reply docs/wordpress.md:1]: yep, I planned to do in next PR, it does not really belong to 'renaming' topic

File: kubernetes/cluster.go (hunk 3 above)
Comment (jsuchome) [reply kubernetes/cluster.go:38]: why not

File: deployments/tekton.go (hunk 4 above)
Comment (jsuchome) [reply deployments/tekton.go:353]: OK, I will move this back but in the end we will probably remove all traefik related stuff

File: docs/wordpress.md (hunk 2 above)
Comment (jsuchome) [reply docs/wordpress.md:1]: there are few other things to remove

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -31,14 +31,14 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: '^1.13.7'
-      - name: Build Carrier
+      - name: Build Fuseml
```
Comment (flaviodsr) [inline .github/workflows/release.yml:34]: I wonder if we could write it `FuseML` instead of `Fuseml` ?

**Review by flaviodsr — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 5 above)
Comment (jsuchome) [reply .github/workflows/release.yml:34]: Here yes. In go code probably not.

File: README.md [hunk 6]
```diff
@@ -43,7 +43,7 @@ What **FuseML** is **NOT**:
 
 ## Inception and Direction
 
-FuseML originated as a fork of our sister open source project [Carrier](https://github.com/SUSE/carrier), a lightweight open source PaaS built on top of Kubernetes, then has been gradually transformed and infused with the MLOps concepts that make it the AI/ML orchestration tool that it is today.
+FuseML originated as a fork of our sister open source project [Fuseml](https://github.com/SUSE/carrier), a lightweight open source PaaS built on top of Kubernetes, then has been gradually transformed and infused with the MLOps concepts that make it the AI/ML orchestration tool that it is today.
```
Comment (stefannica) [inline README.md:46]: How can FuseML be its own sister project ? :smile:

File: docs/wordpress.md (hunk 2 above)
Comment (stefannica) [reply docs/wordpress.md:1]: ok

File: README.md (hunk 6 above)
Comment (jsuchome) [reply README.md:46]: Sorry, I wasn't paying attention

**Review by stefannica — APPROVED** [review]

**Review by flaviodsr — APPROVED** [review]

**Pull request merged**